packages/office-ui-fabric-react/src/components/Dialog/Dialog.tsx

```diff
@@ -16,12 +16,13 @@ import { DialogContent } from './DialogContent';
 @withResponsiveMode
 export class Dialog extends BaseComponent<IDialogProps, {}> {
   public static defaultProps: IDialogProps = {
-    modalProps: {
-      isDarkOverlay: true,
-      isBlocking: false,
-      className: '',
-      containerClassName: ''
-    },
+
+    // modalProps: {
+    //   isDarkOverlay: false,
+    //   isBlocking: false,
+    //   className: '',
+    //   containerClassName: ''
+    // },
```

Comment:
If we do end up needing to remove this, don't forget to remove the comment!

Reply:
Huzzah!